hardhat.config.ts

```diff
@@ -0,0 +1,106 @@
+import { config as dotenvConfig } from "dotenv";
+import { resolve } from "path";
+dotenvConfig({ path: resolve(__dirname, "./.env") });
+import "@nomiclabs/hardhat-etherscan";
+import "hardhat-contract-sizer";
+import { HardhatUserConfig } from "hardhat/config";
+import { NetworkUserConfig } from "hardhat/types";
+import "@nomiclabs/hardhat-waffle";
+import "@nomiclabs/hardhat-ganache";
+import "hardhat-typechain";
+import "solidity-coverage";
+import "./tasks/accounts";
+import "./tasks/clean";
+
+const chainIds = {
+  ganache: 1337,
+  goerli: 5,
+  hardhat: 31337,
+  kovan: 42,
+  mainnet: 1,
+  rinkeby: 4,
+  ropsten: 3,
+};
+
+// Ensure that we have all the environment variables we need.
+let mnemonic: string;
```

Comment:
Easier to read like this
```
const mnemonic = process.env.MNEMONIC;
if (!mnemonic) {
  throw new Error("Please set your MNEMONIC in a .env file");
}
```

Reply:
This comes from https://github.com/paulrberg/solidity-template we should push this request upstream if important